diff --git a/dist/runtime/server/storage.js b/dist/runtime/server/storage.js
index b6663a235cfaa3b00a69564eba62d60bfbbd892d..1c37330fd2393aca0712b38fe1d855d97e5c4ca6 100644
--- a/dist/runtime/server/storage.js
+++ b/dist/runtime/server/storage.js
@@ -82,19 +82,19 @@ export function* chunksFromArray(arr, n) {
   }
 }
 export const getContentsList = /* @__PURE__ */ (() => {
-  let cachedContents = [];
-  let pendingContentsListPromise = null;
-  const _getContentsList = async (event, prefix) => {
-    const keys = await getContentsIds(event, prefix);
-    const keyChunks = [...chunksFromArray(keys, 10)];
-    const contents = [];
-    for (const chunk of keyChunks) {
-      const result = await Promise.all(chunk.map((key) => getContent(event, key)));
-      contents.push(...result);
-    }
-    return contents.filter((c) => c && c._path);
-  };
   return (event, prefix) => {
+    let cachedContents = [];
+    let pendingContentsListPromise = null;
+    const _getContentsList = async (event, prefix) => {
+      const keys = await getContentsIds(event, prefix);
+      const keyChunks = [...chunksFromArray(keys, 10)];
+      const contents = [];
+      for (const chunk of keyChunks) {
+        const result = await Promise.all(chunk.map((key) => getContent(event, key)));
+        contents.push(...result);
+      }
+      return contents.filter((c) => c && c._path);
+    };
     if (event.context.__contentList) {
       return event.context.__contentList;
     }
