diff --git a/dist/runtime/server/storage.js b/dist/runtime/server/storage.js
index b6663a235cfaa3b00a69564eba62d60bfbbd892d..5d96ab27739f75a0217b1d9025ea6e6ce078224b 100644
--- a/dist/runtime/server/storage.js
+++ b/dist/runtime/server/storage.js
@@ -14,22 +14,13 @@ let _sourceStorage;
 let _cacheStorage;
 let _cacheParsedStorage;
 export const sourceStorage = () => {
-  if (!_sourceStorage) {
-    _sourceStorage = prefixStorage(useStorage(), "content:source");
-  }
-  return _sourceStorage;
+  return prefixStorage(useStorage(), "content:source");
 };
 export const cacheStorage = () => {
-  if (!_cacheStorage) {
-    _cacheStorage = prefixStorage(useStorage(), "cache:content");
-  }
-  return _cacheStorage;
+  return prefixStorage(useStorage(), "cache:content");
 };
 export const cacheParsedStorage = () => {
-  if (!_cacheParsedStorage) {
-    _cacheParsedStorage = prefixStorage(useStorage(), "cache:content:parsed");
-  }
-  return _cacheParsedStorage;
+  return prefixStorage(useStorage(), "cache:content:parsed");
 };
 const isProduction = process.env.NODE_ENV === "production";
 const isPrerendering = import.meta.prerender;
@@ -47,9 +38,12 @@ const contentIgnorePredicate = (key) => {
   return true;
 };
 export const getContentsIds = async (event, prefix) => {
+  console.warn('enter in getContentsIds')
   let keys = [];
   if (isProduction) {
+    console.warn('enter in isProduction')
     keys = await cacheParsedStorage().getKeys(prefix);
+    console.warn('keys', keys)
   }
   const source = sourceStorage();
   if (keys.length === 0) {
@@ -85,6 +79,7 @@ export const getContentsList = /* @__PURE__ */ (() => {
   let cachedContents = [];
   let pendingContentsListPromise = null;
   const _getContentsList = async (event, prefix) => {
+    console.warn('enter in _getContentsList')
     const keys = await getContentsIds(event, prefix);
     const keyChunks = [...chunksFromArray(keys, 10)];
     const contents = [];
@@ -116,12 +111,17 @@ export const getContentsList = /* @__PURE__ */ (() => {
 })();
 const pendingPromises = {};
 export const getContent = async (event, id) => {
+  console.warn('enter in getContent')
   const contentId = id;
+  console.warn('contentId', contentId)
   if (!contentIgnorePredicate(id)) {
     return { _id: contentId, body: null };
   }
+  console.warn('after contentIgnorePredicate')
   const source = sourceStorage();
   const cache = cacheParsedStorage();
+  console.warn('source', source)
+  console.warn('cache', cache)
   if (isPreview(event)) {
     const { key } = getPreview(event);
     const previewId = `preview:${key}:${id}`;
